apps/speed.c

```diff
@@ -1369,39 +1369,34 @@ int speed_main(int argc, char **argv)
      * add tests over more curves, simply add the curve NID and curve name to
      * the following arrays and increase the EC_NUM value accordingly.
      */
-    static const unsigned int test_curves[EC_NUM] = {
+    static const struct {
+        const char *name;
+        unsigned int nid;
+        unsigned int bits;
+    } test_curves[] =
+    {
```

Comment:
Hmm, I was under impression that preferred style would be to have opening { at same line. See for example ecp_nistp256.c...

Reply:
It's minor formatting nit, so I'm approving this provided that fix-up would be a separate commit that would be squashed upon commit to repo.

Reply:
Fixed !